server/data/resources/discord-access.json

```diff
@@ -0,0 +1,31 @@
+{
+    "title": "How to Access the MathSoc Discord",
+    "subheader": "Feel free to join and ask any general or academic questions and contact any of the Mathsoc executives and councillors!",
+    "steps": [
+        {
+            "title": "Step 1",
+            "text": "Open the <strong>Discord app</strong> on your computer.",
```

Comment:
With Aryaman's new changes to give us rich text editing in the backend, we're using `<variableName>Markdown` as the key for any data file that includes rich text content. Any field named in that pattern is edited by a rich text editor - you can see this in action by checking out the main branch and visiting http://localhost:3000/admin/editor?page=get-involved/volunteer.

Would you be able to rename the `"text"` field on the steps to something like `"textMarkdown"`, `"descriptionMarkdown"`, etc.?  This will give admins greater ability to edit the steps listed.